main.cpp

```diff
@@ -263,6 +269,10 @@ auto device_selection =
             .if_missing([] { return "missing bus number"; })) % "Filter devices by USB bus number" +
         (option("--address") & integer("addr").min_value(1).max_value(127).set(settings.address)
             .if_missing([] { return "missing address"; })) % "Filter devices by USB device address"
+#if !defined(_WIN32)
+        + option('f', "--force").set(settings.force) % "Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. Unless the command itself causes a reboot, the device will be rebooted back to application mode" +
+        option('F', "--force-no-reboot").set(settings.no_reboot_if_forced) % "Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. Unless the command itself causes a reboot, the device will be left connected and accessible to picotool, but without the RPI-RP2 drive mounted"
```

Comment:
FWIW I still find this "Unless the command itself causes a reboot" text quite confusing - so other users might too? :man_shrugging:

Reply:
> If a Pico is in BOOTSEL mode, and I run `sudo ./picotool reboot -u` it successfully reboots back into BOOTSEL mode, but it also prints `ERROR: Communication with RP2040 device failed` which seems a bit odd?

a yes, i forgot - lost in the mists of time; will fix

> > If I use the -f or -F options on an RP2040 which is already in BOOTSEL mode, I get an error saying "ERROR: Forced command requires a single rebootable RP2040 device to be targeted." which could be slightly confusing. Maybe if the RP2040 is already in BOOTSEL mode it should ignore any 'force' flags?
> 
> I'm still seeing this behaviour on Linux - your previous reply implied this might be changing?

ah yes, i forgot

Reply:
> FWIW I still find this "Unless the command itself causes a reboot" text quite confusing - so other users might too? man_shrugging

I have changed to "After executing the command (unless the command itself is a 'reboot')"